src/main/java/homework_6/map_problems_generator/CollisionClass.java

```diff
@@ -0,0 +1,29 @@
+package homework_6.map_problems_generator;
+
+import lombok.Data;
+import java.util.Calendar;
+
+@Data
+public class CollisionClass {
+    private Calendar dateOfBirth;
+
+    CollisionClass(Calendar dateOfBirth) {
+        this.dateOfBirth = dateOfBirth;
+    }
+
+    @Override
+    public int hashCode() {
+        return 1;
+    }
+
+    @Override
+    public boolean equals(Object obj) {
```

Comment:
I'm really confused why all your methods are public. By this you break one of the OOP principles - incapsulation

Reply:
Oh ((. It's because of the tests. I need to learn to test private methods (or build the architecture correctly).

Reply:
You shouldn't test private methods

Reply:
A-a-ah... I didn't know, I wrote all methods as private, but with the writing tests to them I needed to change them to public.